scripts/checks.sh

```diff
@@ -201,19 +199,17 @@ function test_ipv6_stack()
 		return
 	fi 
 	# Check if we can reach an IPv6 HTTP service since device is configured to do so 
-	local ipv6_check
 	local -i res
-	ipv6_check=$(CURL_CA_BUNDLE=${TMPCRT} ${TIMEOUT_CMD} curl -qs "https://${IPV6_ENDPOINT}")
-	res=$?
+	res=$(CURL_CA_BUNDLE=${TMPCRT} ${TIMEOUT_CMD} curl -qs "https://${IPV6_ENDPOINT}" ; echo $?)
 	if test "$res" != "0"; then
 		# from man curl:
 		# EXIT CODES
 		# 60	 Peer certificate cannot be authenticated with known CA certificates.
 		if [ "${res}" -eq 60 ]; then
-				echo "${FUNCNAME[0]}: There may be a firewall blocking traffic to https://${IPV6_ENDPOINT} (SSL errors)"
-				return
+			echo "${FUNCNAME[0]}: There may be a firewall blocking traffic to https://${IPV6_ENDPOINT} (SSL errors)"
+			return
 		fi
-		echo "${FUNCNAME[0]}: Could not contact https://${IPV6_ENDPOINT}"
+		echo "${FUNCNAME[0]}: Could not contact https://${IPV6_ENDPOINT}. If necessary you can [disable IPv6 on this device](https://www.balena.io/docs/reference/OS/network/2.x/#disable-ipv6)."
```

Comment:
@thgreasi Any idea if this markdown will actually render in the device diagnostics health check results?

Reply:
No I don't think it will @klutchell, since it just uses a plan text component.
See: https://github.com/balena-io/balena-ui/blob/9c2f177548497d2cacc5dc610898cb733b373cff/src/pages/dashboard/device/diagnostics-summary/DeviceChecksSummaryCard.tsx#L86

Reply:
That's kind of what I thought. Do we really need the whole URL printed in the diagnostic report?

Reply:
Would you suggest showing only part of it ? ;)
Given that we hope to reduce support load, I think we need to make it as obvious as possible.
And it's not that this message is going to be shown to all users, if the UX of its size is your concern.

Reply:
> if the UX of its size is your concern.

That was my only concern. In a healthy path it should never be seen.

Reply:
These newlines should render fine though, right? `echo -e` will make sure they are replaced with actual newlines when printed.

Reply:
I don't know @klutchell , but I see that
* we do not do that anywhere else atm
* `run_checks()` is actually using newlines, potentially to separate the results of the tests?? :thinking: 

I guess you better test this on your BoB before merging.